README.markdown

```diff
@@ -101,6 +101,8 @@ To set a default Node version to be used in any new shell, use the alias 'defaul
 
     nvm alias default stable
 
+This creates a symlink to the stable version for the Node or to the verison you can mention instead of stable, so you don't have to `nvm use` everytime you start a shell session.
```

Comment:
`/verison/version`. This seems like a redundant restatement of line 100. I'm not sure it's necessary, but could you adapt line 100 instead of adding a duplicate?